ArduCopter/GCS_Mavlink.h

```diff
@@ -55,6 +55,9 @@ class GCS_MAVLINK_Copter : public GCS_MAVLINK
 
 private:
 
+    // returns false if the input vector (in m/s/s or m/s/s) looks invalid
```

Comment:
let's put the same comment in the .h and .cpp.

Reply:
Done